src/ol/expr/cpu.js

```diff
@@ -200,6 +201,13 @@ function compileAccessorExpression(expression, context) {
     case Ops.Var: {
       return (context) => context.variables[name];
     }
+    case Ops.Concat: {
+      return (context_) => {
+        return "".concat(...expression.args.reduce((a, c) => {
+          return [...a, compileExpression(c, context)(context_).toString()];
+        }, []));
```

Comment:
```suggestion
      const args = expression.args.map((e) => compileExpression(e, context));
      return (context) => args.map((arg) => arg(context)).join('');
```

Compiling of the expression shoukd be done outside the evaluation. This way it will only be compiled once when the style is created.

Reply:
Gotcha, I can move it outside of the context function. But since this uses the concat function as [String.concat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/concat) should I change it to the following?

```javascript
const args = expression.args.map((e) => compileExpression(e, context));
return (context) => ''.concat(...args.map((arg) => arg(context)));
```

I can change it to a 'Join' instead of the concat, but would just need to pop out which argument would be the join's character.

Reply:
I guess the functionality is the same, so either would work.